docs/question/accessibility.md

```diff
@@ -0,0 +1,80 @@
+# Authoring accessible questions
+
+Ensuring that PrairieLearn questions are accessible to all students, including those with disabilities, is crucial for an inclusive learning environment. This guide provides recommendations for authors to create accessible question content.
+
+## Semantic HTML
+
+Using semantic HTML elements appropriately is the foundation of web accessibility. Semantic HTML means using HTML tags for their intended purpose, which helps assistive technologies (like screen readers) understand the structure and meaning of the content.
+
+- **Headings**: Use `<h1>`, `<h2>`, etc., to structure your question content logically. Avoid skipping heading levels.
+- **Lists**: Use `<ol>` for ordered lists and `<ul>` for unordered lists, and `<li>` for list items.
+- **Paragraphs**: Wrap text content in `<p>` tags.
+- **Emphasis**: Use `<em>` for emphasis and `<strong>` for strong importance, rather than relying on visual styling alone.
+
+## Text alternatives for non-text content
+
+All non-text content must have a text alternative that serves an equivalent purpose.
+
+### Images and figures (`pl-figure`)
+
+When using the [`pl-figure`](../elements.md#pl-figure-element) element to embed images, always provide descriptive alternative text (alt text) via the `alt` attribute. Good alt text is concise and conveys the essential information or function of the image.
+
+```html
+<pl-figure
+  file-name="diagram.png"
+  alt="A free-body diagram showing forces acting on a block on an inclined plane."
+></pl-figure>
+```
+
+If the figure contains complex information that's essential for understanding the content, you should provide that information elsewhere in the question text.
```

Comment:
Nit: I think it's important to highlight that the image, whenever necessary, should not be the only source of information necessary to answer a question.

```suggestion
If the figure contains complex information that's essential for understanding the content or to answer the question, you should provide that information elsewhere in the question text.
```

Reply:
Rephrased:

```suggestion
If the figure contains complex information that's essential to understand the content or answer the question, you should provide that information elsewhere in the question text.
```